File: java/src/phase 1-conditional_thinking_java/Q34AmPmConvert.java
import java.util.Scanner;

/**
 * Phase 1: Conditional Thinking (If-Else, Boolean Logic)
 * Level 4: Logical Operators & Compound Statements
 * Question 34: Take 24-hour time (hours and minutes) and print whether it is AM or PM.
 * 
 * Mentor Tips (Java 26):
 * - 0 to 11 is AM, 12 to 23 is PM. (Usually 12:00 PM starts noon, 0:00 AM is midnight).
 */
public class Q34AmPmConvert {

    public static String solve(int hours, int minutes) {
        // TODO: Implement this method to solve the question
        return "";
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Hours (0-23): "); int h = scanner.nextInt();
                System.out.print("Minutes (0-59): "); int m = scanner.nextInt();
                System.out.println("Time: " + solve(h, m));
        scanner.close();
    }
}

File: python/phase 1-condtional_thinking/q34_am_pm_convert.py
"""
Phase 1: Conditional Thinking (If-Else, Boolean Logic)
Level 4: Logical Operators & Compound Statements
Question 34: Take 24-hour time (hours and minutes) and print whether it is AM or PM.

Mentor Tips (Python 3.14+):
- 0 to 11 is AM, 12 to 23 is PM. (Usually 12:00 PM starts noon, 0:00 AM is midnight).
"""

def solve(hours: int, minutes: int) -> str:
    # TODO: Implement this method to solve the question
    return "AM" # or "PM"

def main():
    h = int(input("Hours (0-23): "))\n    m = int(input("Minutes (0-59): "))\n    print(f"Time: {solve(h, m)}")

if __name__ == "__main__":
    main()
